nuxeo-core/nuxeo-core-io/src/main/java/org/nuxeo/ecm/core/io/download/DownloadService.java

```diff
@@ -46,8 +47,14 @@
 
     String NXBIGBLOB = "nxbigblob";
 
+    /** @deprecated since 8.10, use nxtransient instead */
```

Comment:
There is no `nxtransient` defined anywhere. Did you mean to rename NXBIGBLOB to NXTRANSIENT?